app/javascript/src/components/Dashboard/Notes/utils/prettyPrintTime.js

```diff
@@ -0,0 +1,5 @@
+import moment from "moment";
+
+export const calculateAge = timeStamp => moment(timeStamp).fromNow();
+
+export const formatTime = timeStamp => moment(timeStamp).format("ddd, hh:mmA");
```

Comment:
Don't use `moment`.
- This library is dead: https://momentjs.com/docs/#/-project-status/
- The module is unnecessarily bulky.
- Use `dayjs`. Refer: https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=Prefer%20day.js%20over%20inbuilt%20date%20functions

Reply:
Fixed with #22